File: Day 4/Longest Harmonious Subsequence.py
class Solution(object):
    def findLHS(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        count=Counter(nums)
        return max([count[i]+count[i+1] for i in count if i+1 in count] or [0])
        

File: Day 4/Longest Harmonious Subsequence.java
class Solution {
    public int findLHS(int[] nums) {
        HashMap<Integer,Integer> map=new HashMap<Integer,Integer>();
        for(int i=0;i<nums.length;i++){
            if(!map.contains(i)){
                map.put(i,0);
            }
            else{
                map.put(i,map.get(i)++)
            }
        }
        ans=0
        for(int i=0;i<nums.length;i++){
            if(map.contains(i+1)){
               ans=Math.max(ans,) 
            }
        }
    }
}
